tests/conftest.py

```diff
@@ -1190,6 +1190,16 @@ def log_cli_level(pytestconfig):
     return pytestconfig.getini("log_cli_level") or "DEBUG"
 
 
+@pytest.fixture(scope="session", autouse=True)
+def load_cluster_info_file(request):
```

Comment:
If you would like to make sure that this fixture is called before cluster load you should use load_cluster_info_file as a parameter of cluster_load fixture otherwise we will not be sure in which order pytest will load the fixture.

Reply:
Thanks for the note, I reconsidered it and explicitly run the `load_cluster_info` function on the beginning of `login_to_mirror_registry` function (where it was failing because of missing data).